frontend/src/components/core/Sidebar/styled-components.ts

```diff
@@ -59,96 +58,34 @@ export const StyledSidebarNavContainer = styled.div(({ theme }) => ({
 
 export interface StyledSidebarNavItemsProps {
   isExpanded: boolean
-  isOverflowing: boolean
   hasSidebarElements: boolean
 }
 
 export const StyledSidebarNavItems = styled.ul<StyledSidebarNavItemsProps>(
-  ({ isExpanded, isOverflowing, hasSidebarElements, theme }) => {
-    const isExpandedMaxHeight = isExpanded ? "75vh" : "33vh"
-    const maxHeight = hasSidebarElements ? isExpandedMaxHeight : "100vh"
-
+  ({ theme }) => {
     return {
-      maxHeight,
       listStyle: "none",
-      overflow: ["auto", "overlay"],
       margin: 0,
       paddingTop: theme.sizes.sidebarTopSpace,
-      paddingBottom: theme.spacing.lg,
-
-      "&::before": isOverflowing
-        ? {
-            content: '" "',
-            backgroundImage: `linear-gradient(0deg, transparent, ${theme.colors.bgColor})`,
-            width: "100%",
-            height: "2rem",
-            position: "absolute",
-            top: 0,
-            left: 0,
-            right: 0,
-            pointerEvents: "none",
-          }
-        : null,
-
-      "&::after": isOverflowing
-        ? {
-            content: '" "',
-            backgroundImage: `linear-gradient(0deg, ${theme.colors.bgColor}, transparent)`,
-            height: "2rem",
-            position: "absolute",
-            bottom: 0,
-            left: 0,
-            right: 0,
-            pointerEvents: "none",
-          }
-        : null,
     }
   }
 )
 
 export interface StyledSidebarNavSeparatorContainerProps {
   isExpanded: boolean
-  isOverflowing: boolean
 }
```

Comment:
Nit: Same here, looks like the special props are no longer used for styling the `StyledSidebarNavSeparatorContainer`  and the entire interface can be removed.